### ISSUE TITLE
[BUG] Can't run guiscrcpt "No module named 'win32com' "

### ISSUE THREAD

**Issue by rasuyas** [srevinsaju/guiscrcpy#291]
**Describe the bug**
Fresh install of Guiscrcpy using PyPl method. can't run it with  "No module named 'win32com' " Error displayed

**To Reproduce**
1. run `pip3 install -U guiscrcpy`
2. run `guiscrcpy`

**Pip Install Log**
```
>pip3 install -U guiscrcpy
Requirement already satisfied: guiscrcpy in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (4.10.0)
Requirement already satisfied: PyQt5<5.16,>=5.14 in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from guiscrcpy) (5.15.6)
Requirement already satisfied: colorama in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from guiscrcpy) (0.4.4)
Requirement already satisfied: qtpy in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from guiscrcpy) (2.0.0)
Requirement already satisfied: psutil in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from guiscrcpy) (5.9.0)
Requirement already satisfied: cairosvg in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from guiscrcpy) (2.5.2)
Requirement already satisfied: pynput in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from guiscrcpy) (1.7.6)
Requirement already satisfied: click in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from guiscrcpy) (8.0.3)
Requirement already satisfied: PyQt5-sip<13,>=12.8 in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from PyQt5<5.16,>=5.14->guiscrcpy) (12.9.0)
Requirement already satisfied: PyQt5-Qt5>=5.15.2 in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from PyQt5<5.16,>=5.14->guiscrcpy) (5.15.2)
Requirement already satisfied: cssselect2 in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from cairosvg->guiscrcpy) (0.4.1)
Requirement already satisfied: cairocffi in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from cairosvg->guiscrcpy) (1.3.0)
Requirement already satisfied: defusedxml in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from cairosvg->guiscrcpy) (0.7.1)
Requirement already satisfied: pillow in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from cairosvg->guiscrcpy) (9.0.0)
Requirement already satisfied: tinycss2 in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from cairosvg->guiscrcpy) (1.1.1)
Requirement already satisfied: six in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from pynput->guiscrcpy) (1.16.0)
Requirement already satisfied: packaging in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from qtpy->guiscrcpy) (21.3)
Requirement already satisfied: cffi>=1.1.0 in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from cairocffi->cairosvg->guiscrcpy) (1.15.0)
Requirement already satisfied: webencodings in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from cssselect2->cairosvg->guiscrcpy) (0.5.1)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from packaging->qtpy->guiscrcpy) (3.0.6)
Requirement already satisfied: pycparser in c:\users\rasuy\appdata\local\programs\python\python310\lib\site-packages (from cffi>=1.1.0->cairocffi->cairosvg->guiscrcpy) (2.21)
```

**Guiscrcpy runtime error Log**
```
>guiscrcpy
Traceback (most recent call last):
  File "C:\Users\rasuy\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\rasuy\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\rasuy\AppData\Local\Programs\Python\Python310\Scripts\guiscrcpy.exe\__main__.py", line 4, in <module>
  File "C:\Users\rasuy\AppData\Local\Programs\Python\Python310\lib\site-packages\guiscrcpy\cli.py", line 12, in <module>
    from .lib.utils import format_colors as fc, show_message_box
  File "C:\Users\rasuy\AppData\Local\Programs\Python\Python310\lib\site-packages\guiscrcpy\lib\utils.py", line 31, in <module>
    from ..platform.platform import System
  File "C:\Users\rasuy\AppData\Local\Programs\Python\Python310\lib\site-packages\guiscrcpy\platform\platform.py", line 25, in <module>
    from guiscrcpy.platform.windows import Windows as System
  File "C:\Users\rasuy\AppData\Local\Programs\Python\Python310\lib\site-packages\guiscrcpy\platform\windows.py", line 23, in <module>
    from guiscrcpy.platform.windows_tools.tools import make_shortcut
  File "C:\Users\rasuy\AppData\Local\Programs\Python\Python310\lib\site-packages\guiscrcpy\platform\windows_tools\tools.py", line 10, in <module>
    import win32com.client
ModuleNotFoundError: No module named 'win32com'
```

**Desktop:**
* Windows
- **Architecture**: 64-bit
- **Version**: 10

**Additional context**
Python Version 3.101.1 64bit (fresh install)

**Comment by elpekenin** [comment]
I just installed this tool on Windows 10-64 bit and it worked perfect  (python 3.9.7)

1. Installed with `pip install guiscrcpy`
1. Executed with `guiscrcpy`
1. Configured on 1st time by selecting the paths of:
   1.   adb
   1. scrcpy-server
   1. scrcpy.exe 
1. Done, I could already launch it with the big "Start" button

I made a quick search and saw [this](https://superuser.com/questions/609447/how-to-install-the-win32com-python-library)
The module you are missing can be installed by `pip install pypiwin32`
